torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -617,13 +617,15 @@ struct CodeImpl {
       OpCode op;
       if (input->node()->kind() == prim::Constant) {
         op = LOADC;
-      } else if (drop) {
-        op = DROPR;
       } else if (moved) {
         op = MOVE;
       } else {
         op = LOAD;
       }
+
+      if (drop) {
+        op = DROPR;
```

Comment:
It's DROPR, which clears the corresponding register. It does not pop stack. In such case, would the stack still holds the default arg?